MAUI/LinearProgressBar/Animation.md

```diff
@@ -45,19 +45,19 @@ this.Content = linearProgressBar;
 
 ![.NET MAUI Linear ProgressBar with CubicInOut animation](images/animation/easing-animation.gif)
 
-The `SetProgress()` method in the progress bar is used to set progress value along with animation duration and easing effect applicable for the specific method call.
+The [`SetProgress()`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_SetProgress_System_Double_System_Nullable_System_Double__Microsoft_Maui_Easing_) method in the progress bar is used to set progress value along with animation duration and easing effect applicable for the specific method call.
 
 {% highlight c# %}
 
 void SetProgress(double progress, double? animationDuration = null, Easing? easing = null)
 
 {% endhighlight %}
 
-N> The animation duration and easing effect parameters will not affect the configuration of the `AnimationDuration`and `AnimationEasing` properties.
+N> The animation duration and easing effect parameters will not affect the configuration of the [`AnimationDuration`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_AnimationDuration) and [`AnimationEasing`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_AnimationEasing) properties.
 
 ## Indeterminate Easing Effects
 
-The `IndeterminateAnimationEasing` property allows you to specify a transfer function for indeterminate state, which controls animation speed when they run.
+The [`IndeterminateAnimationEasing`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_IndeterminateAnimationEasing) property allows you to specify a transfer function for indeterminate state, which controls animation speed when they run.
 
 The following code sample demonstrates the `BounceIn` easing function of the linear progress bar.
```

Comment:
API link not included.

Reply:
Since its a framework API we cant add the link.